src/templates/organization.js

```diff
@@ -69,16 +47,55 @@ function Tags({ org }) {
     .concat(subCategories)
 
   return (
-    <ul className="my-6">
-      {org.location && <OrganizationLocation text={org.location} />}
-      {org.headcount && (
-        <OrganizationHeadcount text={`${org.headcount} employees`} />
-      )}
-      {org.orgType && <OrganizationOrgType text={org.orgType} />}
+    <SidebarSectionList title="In a snapshot" className="flex flex-col mb-12">
       {categoryList.map(category => (
-        <OrganizationCategory key={category.name} text={category.name} />
+        <SidebarSectionList.Item
+          key={category.name}
+          text={category.name}
+          icon={<FontAwesomeIcon icon={faBox} />}
+        />
       ))}
-    </ul>
+      {org.location && (
+        <SidebarSectionList.Item
+          text={org.location}
+          icon={<FontAwesomeIcon icon={faMapMarkerAlt} />}
+        />
+      )}
+      {org.orgType && (
+        <SidebarSectionList.Item
+          text={org.orgType}
+          icon={<FontAwesomeIcon icon={faBuilding} />}
+        />
+      )}
+      {org.headcount && (
+        <SidebarSectionList.Item
+          text={`${org.headcount} employees`}
+          icon={<FontAwesomeIcon icon={faUsers} />}
+        />
+      )}
+    </SidebarSectionList>
+  )
+}
+
+function SocialLinksSection({ org }) {
+  return (
+    <SidebarSectionList title="Links" className="flex flex-col mb-12">
+      <SidebarSectionList.Link
+        text="Homepage"
+        href={org.homepage}
+        icon={<FontAwesomeIcon icon={faExternalLinkAlt} />}
+      />
+      <SidebarSectionList.Link
+        text={org.title}
+        href={org.linkedIn}
+        icon={<FontAwesomeIcon icon={faLinkedin} />}
+      />
+      <SidebarSectionList.Link
+        text={`@${org.title}`}
```

Comment:
do we know this is true?

We might need to parse the `org.twitter` to get the handle

Reply:
i added a urlPathParser function and also a urlBuilder function.
the parser handles the Twitter display name for now, and may be expanded for other links.
the builder is more a fallback tool for building a new url if it's incorrect. 
for now it takes care of a specific issue i noticed, if you look at "Atlas AI" org page for example, the twitter url that appears in Airtable is "@atlas_ai" without the full url, so Gatsby treat it like a relative path inside our app.
it should be prevented when adding fields to Airtable but for now i fixed it with helper function.